crates/cxx-qt-gen/src/generator/cpp/property/meta.rs

```diff
@@ -4,15 +4,30 @@
 // SPDX-License-Identifier: MIT OR Apache-2.0
```

Comment:
don't forget to add an entry to the CHANGELOG.md :-)

And update anything in the book and we'll have to find somewhere in the properties example of qml_features to use it